auth/codes.md

```diff
@@ -0,0 +1,16 @@
+# Codes
+
+## REJECTED
+
+```sh
+case .userRejectedRequest return 14001
+```
+
+## VALIDATION
```

Comment:
still wondering if those errors really need codes and specs. They will be never send to the peer. 🤔